blog/2023-08-30-magazine-004.md

```diff
@@ -0,0 +1,88 @@
+
+This is an under-the-hood change that should make your code cleaner and your comparisons more precise. No more workarounds or lengthy utility functions to compare complex data structures deeply—they’re baked right in. When you use ==, you can now trust that it’s performing a complete, deep equality check.
+
+
+![deep_equality1](./assets/deep_equality1.png)
+![deep_equality2](./assets/deep_equality2.png)
+
+
+### DateTime Type and Utilities:
+❤️Added by [Tsuf Cohen](https://github.com/tsuf239)❤️
+In our latest update, the DateTime Type along with its associated utilities and methods. Before this enhancement, our system primarily focused on durations. Now, we can offer comprehensive DateTime support, bridging a gap and providing users with a more complete toolset for their time-based operations. This new integration is set to make tasks involving date and time more intuitive and efficient.
+
+![datetime-type](./assets/Datetime_type.png)
+
+### Performance Optimization for JSII Loading: 
+❤️Added by [Yoav Steinberg](https://github.com/yoav-steinberg)❤️ 
+We have also made a significant enhancement in the way we handle JSII libraries. Previously, we relied on a JSON format, which was somewhat sluggish, especially for larger JSII libraries. This shift to a binary format has actually slashed loading times. To put it in perspective, certain libraries that used to take about 4 seconds to load now dazzle with a swift 700-millisecond loading time.
+
+![JSII-module](./assets/JSII_module.png)
+
+### Inferencing for Type Annotations
+
+Taking a stride toward more intuitive coding, our latest update empowers the compiler with the ability to discern certain types, even without the developer spelling it out explicitly. This reduction in verbosity makes for cleaner code. Plans are already underway to further amplify this feature, aiming for even deeper implicit type understanding.
+
+![inference-closure-arguments](./assets/Inference_closure_arguments.png)
+
+
+### Switched Build Tooling to Turbo: 
+
+In an effort to accelerate our processes, we transitioned from Nx to Turbo when it comes to monorepo build tooling and have noticed a surge in performance. Builds that previously had us waiting for 12 minutes, even with caching, have now been supercharged — they wrap up in a mere 2-3 minutes.
+
+
+
+### Multi-File Support: 
+❤️Added by [Chris Rybicki](https://github.com/Chriscbr)❤️
+We’ve been working to enhance and improve Wing’s multi-file support and have just introduced the latest update. Despite an initial challenge that led to a rollback, the updated feature now integrates better with the language server, simplifying the management of multi-file projects.
+
+
+
+### Integrated Console in VSCode: 
+
+We've improved the integration of the console within VSCode, moving beyond the basic iFramed version. This update offers developers better screen utilization. We're also working on further refinements for smoother integration in the future.
+
```

Comment:
The other inflight magazines have had a direct call out thanking all of our contributors. I think we should add that back